mindsdb/integrations/handlers/rag_handler/settings.py

```diff
@@ -106,9 +106,7 @@ def get_available_writer_model_ids(args: dict) -> list:
 def get_available_openai_model_ids(args: dict) -> list:
     """Get available openai LLM model ids"""
 
-    openai.api_key = args["openai_api_key"]
-
-    models = openai.OpenAI().models.list().data
+    models = openai.OpenAI(api_key=args["openai_api_key"]).models.list().data
```

Comment:
before it was using env variable as we needed to update for new openai interface